### ISSUE TITLE
it doesn't work in docker file

### ISSUE THREAD

**Issue by mhamri** [ChristianKohler/PathIntellisense#160]
when I'm writing docker file it would be handy to show the IntelliSense path when referring to a local folder 

```
COPY ./../somewhere/in/my-computer ./somewhere/in/the/docker
```
the first part of the `COPY` command could get benefit of this featuer.

**Comment by ChristianKohler** [comment]
Hey @mhamri , sorry for the very late response.

Did you find a solution for your problem in the meantime?

**Comment by mhamri** [comment]
no, I right click on the file that I want and copy their reference and then modify the path manually.

**Comment by ChristianKohler** [comment]
I tested it out on my machine. So what works is the following. What doesn't work is path completion in non-string paths.

<img width="615" alt="image" src="https://user-images.githubusercontent.com/4374178/143894640-6abd5cd7-902a-4be9-9020-75f2dfb0733d.png">
